app/src/main/java/com/duckduckgo/app/global/api/AtbAndAppVersionPixelRemovalInterceptor.kt

```diff
@@ -55,11 +63,15 @@ class AtbAndAppVersionPixelRemovalInterceptor @Inject constructor() : Intercepto
     private fun isInPixelsList(pixel: String): Boolean {
         return pixels.firstOrNull { pixel.startsWith(it) } != null
     }
+}
 
-    companion object {
-        // list of pixels (pixel name or prefix) for which we'll remove the ATB and App version information
-        @VisibleForTesting
-        internal val pixels = listOf(
+@ContributesMultibinding(
+    scope = AppScope::class,
+    boundType = PixelRequiringDataCleaningPlugin::class,
+)
+object PixelInterceptorPixelsRequiringDataCleaning : PixelRequiringDataCleaningPlugin {
```

Comment:
The plan is that most of these pixels can be moved to another module. However, is also dependent on another branch so will move them after this PR lands.